testing/snapcast/APKBUILD

```diff
@@ -1,70 +1,69 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
-# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
 pkgname=snapcast
-pkgver=0.13.0
+pkgver=0.14.0
 pkgrel=0
 pkgdesc="Synchronous multi-room audio player"
 url="https://github.com/badaix/snapcast"
 arch="all"
 license="GPL-3.0"
-depends="$pkgname-client $pkgname-server"
-makedepends="asio-dev bash avahi-dev flac-dev
-	libvorbis-dev alsa-lib-dev aixlog popl
-	"
-pkgusers="$pkgname"
-pkggroups="$pkgname audio"
-install="$pkgname-client.pre-install $pkgname-server.pre-install"
-subpackages="$pkgname-doc $pkgname-client $pkgname-server"
-source="snapcast-$pkgver.tar.gz::https://github.com/badaix/snapcast/archive/v$pkgver.tar.gz
-	$pkgname-client.initd
-	$pkgname-client.confd
-	$pkgname-server.initd
-	$pkgname-server.confd
-	"
-builddir="$srcdir/snapcast-$pkgver"
+depends="snapserver snapclient"
+makedepends="bash avahi-dev aixlog popl libvorbis-dev flac-dev asio-dev alsa-lib-dev"
+subpackages="$pkgname-doc snapserver snapclient"
+install="snapclient.pre-install snapclient.post-deinstall snapserver.pre-install snapserver.post-deinstall"
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/badaix/$pkgname/archive/v$pkgver.tar.gz
+	snapserver.initd
+	snapclient.initd
+"
 
 build() {
 	cd "$builddir"
 	make
 }
 
-package() {
+check() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" installclient installserver
+	client/snapclient --help > /dev/null
+	server/snapserver --help > /dev/null
 }
 
-client() {
-	pkgdesc="$pkgname client"
-	install_if="$pkgname-$pkgver-r$pkgrel"
-	mkdir -p "$subpkgdir"/usr/bin
-	mv "$pkgdir"/usr/bin/snapclient \
-		"$subpkgdir"/usr/bin/
-	install -Dm755 "$srcdir"/"$subpkgname.initd" \
-		"$subpkgdir"/etc/init.d/"$subpkgname"
-	install -Dm644 "$srcdir"/"$subpkgname.confd" \
-		"$subpkgdir"/etc/conf.d/"$subpkgname"
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" installclient installserver
 }
 
-server() {
+snapserver() {
 	pkgdesc="$pkgname server"
-	install_if="$pkgname-$pkgver-r$pkgrel"
-	mkdir -p "$subpkgdir"/usr/bin
+	depends="avahi"
+
+	cd "$builddir"
+
+	install -dm755 "$subpkgdir"/usr/bin
+	install -dm755 "$subpkgdir"/etc/conf.d
+	install -dm755 "$subpkgdir"/etc/init.d
+
 	mv "$pkgdir"/usr/bin/snapserver \
 		"$subpkgdir"/usr/bin/
-	install -Dm755 "$srcdir"/"$subpkgname.initd" \
-		"$subpkgdir"/etc/init.d/"$subpkgname"
-	install -Dm644 "$srcdir"/"$subpkgname.confd" \
-		"$subpkgdir"/etc/conf.d/"$subpkgname"
+	install -Dm644 server/debian/snapserver.default "$subpkgdir"/etc/default/snapserver
```

Comment:
I am not aware of any other packages using `/etc/default` on Alpine is that location even checked by OpenRC?

Reply:
You're right, it isn't checked by OpenRC, I'll change that in a bit.

I'm not sure if any other packages do it, but like with my other comment, this is where upstream places it so I went with that. I mean, I could probably place it somewhere else, but is there any particular reason against having it there?